tests/server/blueprints/cases/test_cases_views.py

```diff
@@ -377,10 +377,10 @@ def test_case_diagnosis(app, institute_obj, case_obj):
         resp = client.get(url_for("auto_login"))
         assert resp.status_code == 200
 
-        req_data = {"omim_id": "OMIM:615349"}
+        req_data = {"omim_term": "OMIM:615349"}
```

Comment:
Also huh, loosely translated as "why in this PR"? 😄

Reply:
Because that's when I realized that the tests were not correct.

Reply:
:+1:

Reply:
You will be just like Måns before long.. 😉